alembic/versions/2c4af13eef71_migration_message.py

```diff
@@ -0,0 +1,198 @@
+    sa.Column('user_id', sa.Integer(), nullable=True),
+    sa.Column('room_id', sa.Integer(), nullable=True),
+    sa.Column('comments', sa.String(), nullable=True),
+    sa.Column('creation_time', sa.String(), nullable=False),
+    sa.Column('overall_rating', sa.String(), nullable=True),
+    sa.Column('cleanliness_rating', sa.String(), nullable=True),
+    sa.ForeignKeyConstraint(['room_id'], ['rooms.id'], ),
+    sa.ForeignKeyConstraint(['user_id'], ['users.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('resources',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('quantity', sa.Integer(), nullable=False),
+    sa.Column('room_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['room_id'], ['rooms.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('responses',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('room_id', sa.Integer(), nullable=True),
+    sa.Column('question_id', sa.Integer(), nullable=True),
+    sa.Column('rate', sa.Integer(), nullable=True),
+    sa.Column('check', sa.Boolean(), nullable=True),
+    sa.Column('text_area', sa.String(), nullable=True),
+    sa.Column('created_date', sa.DateTime(), nullable=False),
+    sa.ForeignKeyConstraint(['question_id'], ['questions.id'], ),
+    sa.ForeignKeyConstraint(['room_id'], ['rooms.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('devices',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('device_type', sa.String(), nullable=False),
+    sa.Column('date_added', sa.DateTime(), nullable=False),
+    sa.Column('last_seen', sa.DateTime(), nullable=False),
+    sa.Column('location', sa.String(), nullable=False),
+    sa.Column('resource_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['resource_id'], ['resources.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('missing_items',
+    sa.Column('item_id', sa.Integer(), nullable=True),
+    sa.Column('response_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['item_id'], ['resources.id'], ),
+    sa.ForeignKeyConstraint(['response_id'], ['responses.id'], )
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('missing_items')
+    op.drop_table('devices')
+    op.drop_table('responses')
+    op.drop_table('resources')
+    op.drop_table('feedback')
+    op.drop_table('events')
+    op.drop_table('rooms')
+    op.drop_table('wings')
+    op.drop_table('floors')
+    op.drop_table('blocks')
+    op.drop_table('users_roles')
+    op.drop_table('offices')
+    op.drop_table('notifications')
+    op.drop_table('users')
+    op.drop_table('roles')
+    op.drop_table('questions')
+    op.drop_table('locations')
```

Comment:
This alembic file is not necessary, please remove this

Reply:
this has been removed